src/main/resources/config/liquibase/changelog/20210101081000_changelog.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!-- TODO ADD TO MASTER_XML WHEN FEATURE IS FINISHED. FIRST GET OK FROM @STEPHAN KRUSCHE-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="swaldhauser" id="20210101081000-1">
+        <sql>
+            UPDATE exercise SET
+            max_score = 0
+            WHERE max_score IS NULL
+        </sql>
+    </changeSet>
+
+    <changeSet id="20210101081000-2" author="swaldhauser">
+        <addNotNullConstraint tableName="exercise" columnName="max_score" columnDataType="double" defaultNullValue="0"/>
+    </changeSet>
+
+    <changeSet author="swaldhauser" id="20210101081000-3">
+        <sql>
+            UPDATE exercise SET
+            bonus_points = 0
+            WHERE bonus_points IS NULL
+        </sql>
+    </changeSet>
+
+    <changeSet id="20210101081000-4" author="swaldhauser">
+        <addNotNullConstraint tableName="exercise" columnName="bonus_points" columnDataType="double" defaultNullValue="0"/>
+    </changeSet>
+
+    <changeSet id="20210101081000-5" author="swaldhauser">
+        <sql>
+            UPDATE quiz_question qq
+            INNER JOIN exercise e ON qq.exercise_id = e.id
+            SET qq.score = 1
+            WHERE COALESCE(e.max_score,0) = 0
```

Comment:
we should add an additional condition that qq.score was 0 before. It's probably not needed, but it would be more safe

Reply:
Done in e6818b2bc8d9da65a945fe2da76157737ee1dbd4